plutus-playground-client/src/Control/Monad/Error/Extra.purs

```diff
@@ -0,0 +1,17 @@
+module Control.Monad.Error.Extra where
```

Comment:
This code needs to be in https://pursuit.purescript.org/packages/purescript-transformers/3.4.0 because it's been written so many times, perhaps you could open a PR?